container/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:21.7.2-bookworm-slim as frontend-builder
+
+WORKDIR /app
+
+COPY ./frontend/package.json frontend/package-lock.json ./
+RUN npm install
+
+COPY ./frontend ./
+RUN npm run build
+
+
+FROM python:3.12-slim as runtime
+
+WORKDIR /app
+ENV PYTHONPATH '/app'
+ENV RUN_AS_DEVIN=false
+ENV USE_HOST_NETWORK=false
+ENV SSH_HOSTNAME=host.docker.internal
+ENV WORKSPACE_BASE=/opt/workspace_base
+RUN mkdir -p $WORKSPACE_BASE
+
+RUN apt-get update -y
+RUN apt-get install -y curl make git build-essential
```

Comment:
`build-essential` is a pretty large installation, with lots of dependencies that get installed - and does not need to be retained in the runtime image. It's the whole reason I added a separate python-build stage here: https://github.com/foragerr/OpenDevin/pull/10/files#diff-b09b9fb99ae1ede0edf0fe18e34777e67bb38da89272f7af53db8880794437a4R10-R26

Reply:
Good point! I was actually wondering about the python build stage.

Mind if we tackle this in a follow-on?